css/grid.css

```diff
@@ -4,9 +4,12 @@
 }
 
 .row.first-row {
-	padding-top: 35px;
-	padding-left: 45px;
+	padding-top: 10px;
```

Comment:
Not urgent but would be good to generally try to move to use css variables for this.

We have a `--default-grid-baseline` (4px) in the server where any spacing should be a multiple of that one (4/8/12/16/...)

e.g.: padding: `calc(var(--default-grid-baseline) * 3);` for 12px